<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertificateScreen</class>
 <widget class="QWidget" name="CertificateScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>540</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>600</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="certificateLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Congratulations, you have learned how to park!
Please apply this knowledge in your everyday life.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="redoLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Great job completing the final level! To receive your certificate,
return home and complete all levels in order by clicking Start Tutorial.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
